crates/topos/src/components/regtest/mod.rs

```diff
@@ -34,8 +33,13 @@ pub(crate) async fn handle_command(
 
             // Setup instrumentation if both otlp agent and otlp service name
             // are provided as arguments
-            let basic_controller =
-                setup_tracing(verbose, false, cmd.otlp_agent, cmd.otlp_service_name)?;
+            let basic_controller = setup_tracing(
+                verbose,
+                false,
+                cmd.otlp_agent,
+                cmd.otlp_service_name,
+                env!("TOPOS_VERSION"),
```

Comment:
Should this really be a user-controlled value? Shouldn't we set this during build using the `topos` crate version + the git commit hash?

Reply:
`env!` is compile time macro. This is defined in the `build.rs` of the topos crate.

Reply:
🤦